### PR TITLE
Added missing i18n for footer api

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Fixes https://github.com/readthedocs/readthedocs.org/issues/3561.

### CODE REVIEW

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 1]
```diff
@@ -90,14 +91,14 @@
       </dl>
       {% elif bitbucket_url %}
       <dl>
-        <dt>On Bitbucket</dt>
+				<dt>{% trans "On Bitbucket" %}</dt>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]: We should use spaces, no tabs

**Comment by tapaswenipathak** [comment]
@stsewd: Done per review.

**Comment by tapaswenipathak** [comment]
@stsewd: `Search docs` is a placeholder?! You require translation for a placeholder?

**Comment by stsewd** [comment]
@tapaswenipathak yes, don't forget to search more words for translation on the file, I just enumerated a few.

**Comment by tapaswenipathak** [comment]
@stsewd: ✅

**Comment by tapaswenipathak** [comment]
@stsewd: Now see.

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 2]
```diff
@@ -1,11 +1,12 @@
 <!-- Inserted RTD Footer -->
+{% load i18n %}
 <div class="injected">
 
   {% if not new_theme %}
   <div class="rst-versions rst-badge" data-toggle="rst-versions">
     <span class="rst-current-version" data-toggle="rst-current-version">
       <span class="fa fa-book">&nbsp;</span>
-      v: {{ current_version }}
+         {{ current_version }}
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]: This shouldn't be changed

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 3]
```diff
@@ -61,12 +62,12 @@
       {% block readthedocs %}
       <dl>
         <!-- These are kept as relative links for internal installs that are http -->
-        <dt>On Read the Docs</dt>
+        <dt>{% trans "On Read the Docs" %}</dt>
         <dd>
           <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'projects_detail' project.slug %}">Project Home</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]: missing translation

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 4]
```diff
@@ -61,12 +62,12 @@
...
         </dd>
         <dd>
-          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'builds_project_list' project.slug %}">Builds</a>
+          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'builds_project_list' project.slug %}">{% trans "Builds" %}</a>
         </dd>
         <dd>
           <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'project_downloads' project.slug %}">Downloads</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]: missing translation

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 5]
```diff
@@ -112,11 +113,11 @@
 
       {% block search %}
       <dl>
-        <dt>Search</dt>
+        <dt>{% trans "Search" %}</dt>
         <dd>
           <div style="padding: 6px;">
             <form id="flyout-search-form" class="wy-form" target="_blank" action="//{{ settings.PRODUCTION_DOMAIN }}{% url 'elastic_project_search' project.slug %}" method="get">
-              <input type="text" name="q" placeholder="Search docs">
+              <input type="text" name="q" placeholder="{% trans "Search docs" %}>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]: Missing `"` at the end

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 6]
```diff
@@ -128,9 +129,9 @@
       <hr/>
       {% block footer %}
         <small>
-          <span>Hosted by <a href="https://readthedocs.org">Read the Docs</a></span>
+          <span>{% trans "Hosted by" %}<a href="https://readthedocs.org">{% trans "Read the Docs" %}</a></span>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]: Read the Docs is the company name, we shouldn't translate that, also, probably you want to use a blocktrans like https://github.com/stsewd/readthedocs.org/blob/8c2f8e77731dfc53149f3baa791abe080a9809a3/readthedocs/templates/projects/project_translations.html#L23-L27

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 7]
```diff
@@ -128,9 +129,9 @@
...
           <span> &middot; </span>
-          <a href="https://docs.readthedocs.io/page/privacy-policy.html">Privacy Policy</a>
+            <a href="https://docs.readthedocs.io/page/privacy-policy.html">{% trans "Privacy Policy" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]: Previous indentation was fine

File: readthedocs/api/v2/templates/restapi/footer.html (hunk 2 above)
Comment (tapaswenipathak) [reply readthedocs/api/v2/templates/restapi/footer.html]: You are seeing old edit - https://i.gyazo.com/bca5cf1cbf6cff25da2603cbed0a5b4a.png.

Comment (stsewd) [reply readthedocs/api/v2/templates/restapi/footer.html]: The diff is showing that `v:` was removed, that shouldn't be removed.

File: readthedocs/api/v2/templates/restapi/footer.html (hunk 6 above)
Comment (tapaswenipathak) [reply readthedocs/api/v2/templates/restapi/footer.html]: Right, ok.

The difference
- blocktrans: 
https://docs.djangoproject.com/en/dev/topics/i18n/translation/#std:templatetag-blocktrans
- trans: https://docs.djangoproject.com/en/dev/topics/i18n/translation/#std:templatetag-trans

I think the existing tagging is best. If there is anything other than `href` or text, we can use blocktrans. Useful summary: https://stackoverflow.com/a/17621832.

File: readthedocs/api/v2/templates/restapi/footer.html (hunk 2 above)
Comment (tapaswenipathak) [reply readthedocs/api/v2/templates/restapi/footer.html]: I ignored that, fixed.

**Comment by tapaswenipathak** [comment]
Thanks @stsewd, please see now.

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 8]
```diff
@@ -61,15 +62,15 @@
       {% block readthedocs %}
       <dl>
         <!-- These are kept as relative links for internal installs that are http -->
-        <dt>On Read the Docs</dt>
+        <dt>{% trans "On Read the Docs" %}</dt>
         <dd>
-          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'projects_detail' project.slug %}">Project Home</a>
+         <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'projects_detail' project.slug %}">{% trans "Project Home" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'projects_detail' project.slug %}">{% trans "Project Home" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 9]
```diff
@@ -61,15 +62,15 @@
...
         </dd>
         <dd>
-          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'builds_project_list' project.slug %}">Builds</a>
+         <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'builds_project_list' project.slug %}">{% trans "Builds" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'builds_project_list' project.slug %}">{% trans "Builds" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 10]
```diff
@@ -61,15 +62,15 @@
...
         </dd>
         <dd>
-          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'project_downloads' project.slug %}">Downloads</a>
+         <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'project_downloads' project.slug %}">{% trans "Downloads" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'project_downloads' project.slug %}">{% trans "Downloads" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 11]
```diff
@@ -78,32 +79,32 @@
 
       {% if github_edit_url %}
       <dl>
-        <dt>On GitHub</dt>
+        <dt>{% trans "On GitHub" %}</dt>
         <dd>
-          <a href="{{ github_view_url }}">View</a>
+         <a href="{{ github_view_url }}">{% trans "View" }</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="{{ github_view_url }}">{% trans "View" }</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 12]
```diff
@@ -78,32 +79,32 @@
...
         </dd>
         {% if version.is_editable %}
         <dd>
-          <a href="{{ github_edit_url }}">Edit</a>
+         <a href="{{ github_edit_url }}">{% trans "Edit" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="{{ github_edit_url }}">{% trans "Edit" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 13]
```diff
@@ -78,32 +79,32 @@
...
         </dd>
         {% endif %}
       </dl>
       {% elif bitbucket_url %}
       <dl>
-        <dt>On Bitbucket</dt>
+        <dt>{% trans "On Bitbucket" %}</dt>
         <dd>
-          <a href="{{ bitbucket_url }}">Edit</a>
+         <a href="{{ bitbucket_url }}">{% trans "Edit" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="{{ bitbucket_url }}">{% trans "Edit" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 14]
```diff
@@ -78,32 +79,32 @@
...
         </dd>
       </dl>
       {% elif gitlab_edit_url %}
       <dl>
-        <dt>On GitLab</dt>
+        <dt>{% trans "On GitLab" %}</dt>
         <dd>
-          <a href="{{ gitlab_view_url }}">View</a>
+         <a href="{{ gitlab_view_url }}">{% trans "View" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="{{ gitlab_view_url }}">{% trans "View" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 15]
```diff
@@ -78,32 +79,32 @@
...
         </dd>
         {% if version.is_editable %}
         <dd>
-          <a href="{{ gitlab_edit_url }}">Edit</a>
+         <a href="{{ gitlab_edit_url }}">{% trans "Edit" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="{{ gitlab_edit_url }}">{% trans "Edit" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 16]
```diff
@@ -128,9 +129,9 @@
       <hr/>
       {% block footer %}
         <small>
-          <span>Hosted by <a href="https://readthedocs.org">Read the Docs</a></span>
+        <span>{% trans "Hosted by" %}<a href="https://readthedocs.org">Read the Docs</a></span>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
         <span>{% trans "Hosted by" %}<a href="https://readthedocs.org">Read the Docs</a></span>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 17]
```diff
@@ -128,9 +129,9 @@
...
           <span> &middot; </span>
-          <a href="https://docs.readthedocs.io/page/privacy-policy.html">Privacy Policy</a>
+           <a href="https://docs.readthedocs.io/page/privacy-policy.html">{% trans "Privacy Policy" %}</a>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <a href="https://docs.readthedocs.io/page/privacy-policy.html">{% trans "Privacy Policy" %}</a>
```

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 18]
```diff
@@ -128,9 +129,9 @@
       <hr/>
       {% block footer %}
         <small>
-          <span>Hosted by <a href="https://readthedocs.org">Read the Docs</a></span>
+         <span>{% trans "Hosted by" %}<a href="https://readthedocs.org">Read the Docs</a></span>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <span>{% trans "Hosted by" %}<a href="https://readthedocs.org">Read the Docs</a></span>
```

**Comment by tapaswenipathak** [comment]
I didn't change the indentation; editor created issues. Thanks for doing the corrective edits.

File: readthedocs/api/v2/templates/restapi/footer.html [hunk 19]
```diff
@@ -128,9 +129,9 @@
       <hr/>
       {% block footer %}
         <small>
-          <span>Hosted by <a href="https://readthedocs.org">Read the Docs</a></span>
+          <span>{% trans "Hosted by" %}<a href="https://readthedocs.org">Read the Docs</a></span>
```
Comment (stsewd) [inline readthedocs/api/v2/templates/restapi/footer.html]:
```suggestion
          <span>{% trans "Hosted by" %} <a href="https://readthedocs.org">Read the Docs</a></span>
```

Comment (stsewd) [reply readthedocs/api/v2/templates/restapi/footer.html]: We don't want these together

Comment (stsewd) [reply readthedocs/api/v2/templates/restapi/footer.html]: Sorry I missed this on the last review

**Pull request merged**